src/components/OnTheDaySection/UrgentAnnouncement/index.tsx

```diff
@@ -7,7 +7,14 @@ import { LinkText } from "@/components/OnTheDaySection/Notes/LinkText";
  */
 export function UrgentAnnouncement() {
   return (
```

Comment:
[nits] propsで表示非表示を切り替えられるようにすると楽かなと思いました！
```suggestion
export function UrgentAnnouncement({
  isVisible,
}: {
  /**コンポーネントを表示するかどうか */
  isVisible: boolean;
}) {
  if (!isVisible) return null;

  return (
```